src/screens/OtherProfile/index.js

```diff
@@ -540,14 +632,14 @@ const OtherProfile = () => {
           data={handleDataFeed()}
           onScroll={handleScroll}
           onRefresh={handleRefresh}
-          refreshing={isLoading}
+          refreshing={loading}
           onMomentumScrollEnd={(event) => {
             onWillSendViewPostTime(event, feeds);
             fetchNextFeeds(event);
           }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The loading state is now determined by `loading` instead of `isLoading`. Also, the `fetchNextFeeds` and `onWillSendViewPostTime` functions are called when the momentum scroll ends. Ensure that these functions correctly update the state and UI to reflect the updated feeds and post view time.
```diff
+ refreshing={loading}
+ onMomentumScrollEnd={(event) => {
+   onWillSendViewPostTime(event, feeds);
+   fetchNextFeeds(event);
+ }}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->